docs/UpgradeGuide2.md

```diff
@@ -40,18 +41,33 @@ Outlines the changes for our instrumentations.
 
 <h2 id="2.0-basic-usage">Basic Usage</h2>
 
+<h3 id="2.0-gem-rename">Rename to `datadog` gem</h3>
+
+The gem now includes new capabilities (Profiling, Application Security Monitoring) that extend beyond tracing. To better reflect the broad set of features already within this package, and our intention for this package to be an all-inclusive offering, the existing gem has been renamed to `datadog`.
+
+This name change is does not reflect any material change to existing capabilities in 2.0, but future 2.x versions may include additional capabilities or products.
+
+Make sure to update Gemfile:
+
+```ruby
+# === This is a shim to help migrating to the new `datadog` gem ===
```

Comment:
What part of it is the `shim` here?

Reply:
I am preparing. https://github.com/DataDog/dd-trace-rb/pull/3586